app/src/main/java/com/example/footprnt/Map/FeedActivity.java

```diff
@@ -41,13 +36,22 @@
     private SwipeRefreshLayout mSwipeContainer;
     private double mLat;
     private double mLong;
-    private View menu;
+    private View mMenu;
     private TextView mNoPosts;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_feed);
+        initialization();
+        handleMenuAction();
+        handleSwipeRefresh();
+    }
+
+    /**
+     * Initializes all variables
+     */
+    private void initialization(){
         mLat = getIntent().getExtras().getDouble("latitude");
```

Comment:
One thing that's always good to do is to make sure your code won't crash. Right now there's the potential that getIntent().getExtras() could be null so I'd do a check for that before setting these variables just to be safe.